lib/WP_Auth0_LoginManager.php

```diff
@@ -587,6 +576,68 @@ public function end_session() {
 		}
 	}
 
+	/**
+	 * Get and filter the scope used for access and ID tokens
+	 *
+	 * @param string $context - how are the scopes being used?
+	 *
+	 * @return string
+	 */
+	public static function get_userinfo_scope( $context = '' ) {
+		return apply_filters( 'auth0_auth_token_scope', 'openid email name nickname picture', $context );
+	}
+
+	/**
+	 * Get a base authorize URL for handling HLP redirects.
+	 *
+	 * @param null|string $connection - specific connection to use, pass null to use all enabled.
+	 * @param null|string $redirect_to - URL to redirect upon successful authentication.
+	 *
+	 * @return array
+	 */
+	public static function get_authorize_params( $connection = null, $redirect_to = null ) {
+		$params = array();
+		$options = WP_Auth0_Options::Instance();
+		$lock_options = new WP_Auth0_Lock10_Options();
+		$is_implicit = (bool) $options->get( 'auth0_implicit_workflow', FALSE );
+		$nonce = WP_Auth0_Nonce_Handler::getInstance()->get();
+
+		$params[ 'client_id' ] = $options->get( 'client_id' );
+		$params[ 'scope' ] = self::get_userinfo_scope( 'authorize_url' );
+		$params[ 'response_type' ] = $is_implicit ? 'id_token': 'code';
```

Comment:
I think this value should be changeable by the user. You can also [accept combinations](https://openid.net/specs/oauth-v2-multiple-response-types-1_0.html#MultiValueResponseTypes) like `id_token code`

Reply:
True but the plugin doesn't handle that so someone would have to build their own entire auth callback to handle.

Reply:
I see what you're saying here but I don't think we need to include functionality now that we're not sure we're going to use later. It would just be a matter of adding a parameter now or later.